docs/data/toolpad/connecting-to-data-sources/fetch.md

```diff
@@ -0,0 +1,34 @@
+# Fetch
+
+<p class="description">Fetch query type is the easiest and fastest way to load external data within Toolpad app.</p>
+
+## Working with fetch
```

Comment:
This section has become a bit outdated due to [recent changes](https://github.com/mui/mui-toolpad/pull/721) to the fetch datasource.

I'm fine with merging as is and revisiting this as follow-up

Reply:
Yeah, let's merge as is as I'm now afraid it can get even more out-dated. I can fix this up in a follow up